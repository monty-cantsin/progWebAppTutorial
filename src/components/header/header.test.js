import React from 'react'
import ReactDOM from 'react-dom'
import ReactTestRender from 'react-test-renderer'
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'

import Header from './header'

test('Header component should render as expected', () => {
	const component = shallow(<Header />)
	const tree = toJson(component)

	expect(tree).toMatchSnapshot()
})