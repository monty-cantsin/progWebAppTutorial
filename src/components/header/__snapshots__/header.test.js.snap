// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render as expected 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <h1
      className="title"
    >
      Taylor
    </h1>
    <nav>
      <ul
        className="list"
      >
        <li
          className="listItem"
        >
          <Link
            activeClassName=""
            activeOnlyWhenExact={false}
            activeStyle={Object {}}
            className="link"
            isActive={[Function]}
            replace={false}
            style={Object {}}
            to="/"
          >
            Posts
          </Link>
        </li>
        <li
          className="listItem"
        >
          <Link
            activeClassName=""
            activeOnlyWhenExact={false}
            activeStyle={Object {}}
            className="link"
            isActive={[Function]}
            replace={false}
            style={Object {}}
            to="/about"
          >
            About
          </Link>
        </li>
      </ul>
    </nav>
  </div>
</header>
`;
